app/models/order_cycle.rb

```diff
@@ -150,6 +156,23 @@ def self.earliest_closing_times
     ]
   end
 
+  def attachable_payment_methods
+    Spree::PaymentMethod.available(:both).
+      joins("INNER JOIN distributors_payment_methods
+             ON payment_method_id = spree_payment_methods.id").
```

Comment:
Would it be better to create an association for this?

Reply:
We could `joins(:distributors)`? Although then the query joins two tables but that probably not a big deal. There is no `id` column on the `distributors_payment_methods` table so not sure if there are other options.